launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/ConnectivityManager.java

```diff
@@ -76,19 +75,23 @@ class ConnectivityManager {
     // data is stored; 2. to implement additional logic that does not depend on what kind of data
     // source we're using, like "if there was an error, update the ConnectionInformation."
     private class DataSourceUpdateSinkImpl implements DataSourceUpdateSink {
+        private final ContextDataManager contextDataManager;
```

Comment:
For reviewers:  The sink now holds the reference to the data manager instead of a private class holding a reference to an enclosing class's member.